DataGenerator/DataGenerator.py

```diff
@@ -18,180 +18,222 @@
 from Utils.GenerateSmoothLabel import get_smoothed_label_distribution
 from sklearn.preprocessing import StandardScaler
 import xnat
+import SimpleITK as sitk
+from monai.data import image_reader
+from monai.transforms import LoadImage, LoadImaged
+
 
 class DataGenerator(torch.utils.data.Dataset):
-    def __init__(self, PatientList, label, config, keys, inference=False, n_norm = None, c_norm = None, transform=None, target_transform = None):
+    def __init__(self, PatientList, transform=None, **kwargs):
         super().__init__()
-        self.keys             = list(keys)
-        self.transform        = transform
-        self.target_transform = target_transform
-        self.label            = label
-        self.inference        = inference
-        self.PatientList      = PatientList
-        self.n_norm = n_norm
-        self.c_norm = c_norm
-        self.config = config
+        self.transform = transform
+        self.config = kwargs['config']
```

Comment:
You shouldn't use the kwargs argument here. You should define config in the function declaration